jest.setup.ts

```diff
@@ -0,0 +1,24 @@
+class MockAudio {
+  public isPlaying = false;
+
+  public volume = 1;
+
+  public currentTime = 0;
+
+  public duration = 0;
+
+  play() {
+    this.isPlaying = true;
+  }
+
+  pause() {
+    this.isPlaying = false;
+  }
+
+  fastSeek(time: number) {
+    this.currentTime = time;
+  }
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+(globalThis as any).Audio = MockAudio;
```

Comment:
This is not ideal because if you added another test file accessing Audio and you run jest on a multithreaded machine you could get two test files modifying the same MockAudio properties simultaneously. Can be avoided by moving this into the individual test file I think